antora-ui-camel/src/css/nav.css

```diff
@@ -294,3 +294,10 @@ html.is-clipped--nav {
   background-position-y: 10px;
   width: 11rem;
 }
+
+aside.nav .nav-menu li a:hover,
+aside.nav .nav-menu .nav-text:hover {
+  color: #ed8225 !important;
+  text-decoration: underline;
+  text-decoration-color: #ed8225 !important;
+}
```

Comment:
This needs to be next to the .nav-menu on lines 118-146, needs to use the CSS variables we have for colors and needs to be without the `!important`.

Reply:
Ok, thanks for the input. I will fix it.